lake/index/filter.go

```diff
@@ -2,15 +2,31 @@ package index
 
 import (
 	"context"
+	"math"
 
 	"github.com/brimdata/zed"
 	"github.com/brimdata/zed/compiler/ast/dag"
+	zedexpr "github.com/brimdata/zed/expr"
+	"github.com/brimdata/zed/expr/extent"
+	"github.com/brimdata/zed/field"
 	"github.com/brimdata/zed/index"
+	"github.com/brimdata/zed/order"
 	"github.com/brimdata/zed/pkg/storage"
 	"github.com/segmentio/ksuid"
+	"go.uber.org/multierr"
 	"golang.org/x/sync/semaphore"
 )
 
+var (
+	minExpr = zedexpr.NewDotExpr(field.Dotted("seek.min"))
+	maxExpr = zedexpr.NewDotExpr(field.Dotted("seek.max"))
+	MaxSpan = extent.NewGenericFromOrder(
+		zed.NewUint64(0),
+		zed.NewUint64(math.MaxUint64),
+		order.Asc,
+	)
```

Comment:
Nit: We don't usually wrap after each argument.
```suggestion
	MaxSpan = extent.NewGenericFromOrder(zed.NewUint64(0), zed.NewUint64(math.MaxUint64), order.Asc)
```